scripts/result_scanner/test/test_result_scanner.rb

```diff
@@ -0,0 +1,140 @@
+require 'minitest/autorun'
+require_relative '../result_scanner'
+
+# For now, you should just be able to `ruby test_result_scanner.rb`
+
+class TestResultScanner < MiniTest::Test
+
+  def setup
+    @last_run_path = "#{__dir__}/fixtures/last_run"
+    @current_run_path = "#{__dir__}/fixtures/current_run"
+    @last_json = ResultScanner.read_json("#{@last_run_path}/fake-site-1.json")
+    @current_json = ResultScanner.read_json("#{@current_run_path}/fake-site-1.json")
+    @last_results = ResultScanner.create_results("fake-site-1", @last_json)
+    @current_results = ResultScanner.create_results("fake-site-1", @current_json)
+    @deltas = ResultScanner.compute_risk_level_deltas(@last_results, @current_results)
+  end
+
+  def test_a_full_result_set
+    ResultScanner.summarize_results("#{__dir__}/fixtures/last_full_mp", "#{__dir__}/fixtures/current_full_mp", __dir__)
```

Comment:
I'd suggest using the block form of [`Dir.mktmpdir`](http://ruby-doc.org/stdlib-2.0.0/libdoc/tmpdir/rdoc/Dir.html#method-c-mktmpdir) for the `output_path` so that it does the cleanup automatically.